api/v2/checluster_types.go

```diff
@@ -103,24 +103,24 @@ type CheClusterComponents struct {
 	Metrics ServerMetrics `json:"metrics"`
 }
 
-// Configuration settings related to the Networking used by the Che installation.
+// Configuration settings related to the networking used by the Che installation.
 // +k8s:openapi-gen=true
 type CheClusterSpecNetworking struct {
-	// Defines labels which will be set to an ingress (a route for OpenShift platform).
+	// Defines labels which will be set for an Ingress (a route for OpenShift platform).
 	// +optional
 	Labels map[string]string `json:"labels,omitempty"`
-	// Defines annotations which will be set to an ingress (a route for OpenShift platform).
+	// Defines annotations which will be set for an Ingress (a route for OpenShift platform).
 	// The defaults for kubernetes platforms are:
 	//     kubernetes.io/ingress.class:                       "nginx"
 	//     nginx.ingress.kubernetes.io/proxy-read-timeout:    "3600",
 	//     nginx.ingress.kubernetes.io/proxy-connect-timeout: "3600",
 	//     nginx.ingress.kubernetes.io/ssl-redirect:          "true"
 	// +optional
 	Annotations map[string]string `json:"annotations,omitempty"`
-	// For an OpenShift cluster, the Operator uses the domain to generate a hostname for a route.
+	// For an OpenShift cluster, the operator uses the domain to generate a hostname for the route.
```

Comment:
From the [SSG wordlist for the letter O](https://redhat-documentation.github.io/supplementary-style-guide/#_o): The term "Operator" in the context of Kubernetes is always capitalized to distinguish it from other types of operators, such as human or mathematical operators.

Reply:
This PR contains multiple instances of "operator."

Reply:
I see. I thought it's capitalized only when it's a part of a full name ( XYZ Operator). Let me fix it.